src/Components/ImagesTable/Target.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Skeleton } from '@patternfly/react-core';
+
+import {
+  ImageTypes,
+  useGetComposeClonesQuery,
+} from '../../store/imageBuilderApi';
+import { ComposesResponseItem } from '../../store/imageBuilderApi';
+
+const targetOptions: { [key in ImageTypes]: string } = {
+  aws: 'Amazon Web Services',
+  azure: 'Microsoft Azure',
+  'edge-commit': 'Edge Commit',
+  'edge-installer': 'Edge Installer',
+  gcp: 'Google Cloud Platform',
+  'guest-image': 'Virtualization - Guest image',
+  'image-installer': 'Bare metal - Installer',
+  vsphere: 'VMWare vSphere',
+  'vsphere-ova': 'VMWare vSphere',
+  wsl: 'Windows Subsystem for Linux',
+  ami: 'Amazon Web Services',
+  'rhel-edge-commit': 'RHEL Edge Commit',
+  'rhel-edge-installer': 'RHEL Edge Installer',
+  vhd: '',
```

Comment:
Is the string empty, because we don't use vhd? If so, then it's perfectly fine.

Reply:
I don't think we use it. I suppose we could have it read 'Virtual Hard Disk'? That's not a bad idea... I just added it to satisfy Typescript's requirement that there is a key for every image type.

Reply:
Right, we don't as far as I know. We could fill it in, but I'm also not opposed to leaving it blank to signify that we don't really work with vhd.